ORACLES/db01_sta.json

```diff
@@ -0,0 +1,145 @@
+/* SALENA 2 July 2023         Comments about changes that I've made
```

Comment:
JSON doesn't allow comments, so this is not legal JSON.  I think it would be desirable for a file that has a ".json" extension to be legal JSON.  Would it be possible to either remove comments or else move JSON-plus-comments files to a `md` format?